app/models.py

```diff
@@ -2566,6 +2566,12 @@ class Mailbox(Base, ModelMixin):
         sa.Boolean, default=False, nullable=False, server_default="0"
     )
 
+    # smime
+    smime_public_key = sa.Column(sa.Text, nullable=True)
```

Comment:
Can you move this into a different model like `MailboxSMimeKey` ? We load mailboxes all the time from the db and the lighter they are, the better. We only need to load the key when we're about to use it.